docs/source/guides/libraries/javascript.rst

```diff
@@ -0,0 +1,134 @@
+Javascript library
+-----------------
+
+.. image:: https://img.shields.io/npm/v/iroha-helpers.svg
+
+This library provides functions which will help you to interact with Hyperledger Iroha from your JS program.
+
+Installation
+^^^^^^^^^^^^
+Via npm
+
+.. code:: sh
+
+    $ npm i iroha-helpers
+
+Via yarn
+
+.. code:: sh
+
+    $ yarn add iroha-helpers
+
+Commands
+^^^^^^^^
+For usage of any command you need to provide ``commandOptions`` as a first argument.
+
+.. code-block:: javascript
+
+  const commandOptions = {
+    privateKeys: [''], // Array of private keys in hex format
```

Comment:
I suggest putting an example of a key here. It may not be clear for the reader what kind of string is expected here.

An example will implicitly show the alphabet, the length.